js/actors/player.js

```diff
@@ -29,6 +30,7 @@ class Player extends Actor {
         this.updateVelocity();
         this.updatePosition();
         this.handleCollisionsWithSolids(this.isSwinging());
+        this.arm.act();
```

Comment:
arm should be added to level.actors so that arm.act() is called in level rather than here

Reply:
am now seeing why that is difficult. i added a card in trello to remove references to levels in actors, and instead pass in the level through actor.act() and actor.animate()